### PR TITLE
Website: Add file type icons page

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Split Brand icons page into Brand Icons and File Type Icons pages.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10355)

### CODE REVIEW

File: apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+To use the file type icons, select the format and size that best meets your needs. The following code shows you how to reference the file. Be sure to use the right resolution for the pixel density of the screen you’re targeting.
```
Comment (Jahnp) [inline apps/fabric-website/src/pages/Styles/FileTypeIconsPage/docs/web/FileTypeIconsImplementation.md]: I found this section a bit challenging to parse, so I took a crack at a rewrite. Take a look and let me know what you think:

---
The most durable and future-proof way to get the right icon for a file or item is to use the [file-type-icons package from Fabric](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/file-type-icons) ([`@uifabric/file-type-icons`](https://www.npmjs.com/package/@uifabric/file-type-icons) on npm). 

The following code shows you how to specify a file type icon by extension, item type, icon size, and image type using the `file-type-icons` package along with Fabric's [`<Icon>` component](https://developer.microsoft.com/en-us/fabric#/controls/web/icon):

```tsx
// Import and initialize file type icons at the app level
import { initializeFileTypeIcons } from '@uifabric/file-type-icons';
initializeFileTypeIcons(/* optional base url */);

// Use icons
import { Icon } from 'office-ui-fabric-react';
import { getFileTypeIconProps, FileIconType } from '@uifabric/file-type-icons';

<Icon {...getFileTypeIconProps({ extension: 'docx', size: 96, imageFileType: 'png' }) } />
<Icon {...getFileTypeIconProps({ type: FileIconType.folder, size: 20, imageFileType: 'svg' }) } />
```

File type icons come in predefined dimensions at 16px, 48px, and 96px. When specifying `size`, stick to these default sizes so the images appear as intended. When specifying image file format (`imageFileType`) and `size`, consider the screen size and pixel density of the screen you're targeting. Note that Fabric Core includes custom media queries that will automatically display the PNG icons in a different resolution depending on the device's pixel density.

### References
Here's a [simple demo](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/src/components/FileTypeIcon/examples/FileTypeIcon.Basic.Example.tsx) of how to use the package. The code for the main method you'll use ([`getFileTypeIconProps`](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/file-type-icons/src/getFileTypeIconProps.ts)) is also well-commented. The icons are kept up-to-date on Fabric's CDN, so we recommend not copying or reference these assets directly.

#### Map icons to extensions - [TypeScript mapping file](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/file-type-icons/src/FileTypeIconMap.ts)

This file maps file extensions to the right PNG or SVG icon. The mapping minimizes the number of required icons while maximizing the number of files that get a non-generic icon. Support for non-file system objects which may not have file extensions can be added in [getFileTypeIconProps.ts](https://github.com/OfficeDev/office-ui-fabric-react/blob/ec9cecc9bf3e3e36c774b43e39bad2223d182358/packages/file-type-icons/src/getFileTypeIconProps.ts#L56) and [FileIconType.ts](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/file-type-icons/src/FileIconType.ts).

**Comment by maxwellred** [comment]
Hi @marygans when you get a chance, can you take a peak at Peter's feedback, thanks!

**Comment by maxwellred** [comment]
LGTM, @ecraig12345 I was hoping for you to have a look and approve when you get the chance, thanks!

**Comment by ecraig12345** [comment]
Looks fine to me too but I think @Jahnp should approve since he had feedback before.

**Comment by marygans** [comment]
@bigbadcapers as well 😊

**Comment by bigbadcapers** [comment]
changes look good! i'm not a formal approver on this repo but you have my Yes in spirit 🤣

**Pull request merged**